src/resource/auth/test/auth.service.spec.ts

```diff
@@ -39,12 +43,18 @@ describe('AuthService should be...', () => {
   });
 
   it.todo('able to validate user');
+
   it('able to login', async () => {
     const user = mockUsers[0];
-    await expect(service.login(user)).resolves.toEqual({
+    expect(service.login(user)).resolves.toEqual({
       user,
     });
   });
-  it.todo('able to register');
-  it.todo('able to request password reset email');
+
+  it('able to register', async () => {
+    const { email, username, password, acceptPolicy } = mockUsers[1];
+    expect(
+      service.register({ email, username, password, acceptPolicy }),
+    ).resolves.toEqual(undefined);
```

Comment:
Why should `service.register` returns `undefined`?

And you can do `expect(await service.register({})` instead of using `.resolves`

Reply:
I'm not sure how this test should be written, the reason I put `undefined` here is because of this:
<img width="1504" alt="Screen Shot 2022-06-24 at 4 19 34 pm" src="https://user-images.githubusercontent.com/89491470/175474775-d3fddb0d-3afa-4142-a1d3-6d6fd787a8dd.png">

I've removed this test for service.register, and updated to todo.